backend-project/setup.cfg

```diff
@@ -5,3 +5,4 @@ exclude = */migrations/*,./bower_components/*,./node_modules/*
 max-complexity = 10
 per-file-ignores =
     ./backend-project/config/settings/*.py: F405,E501,W391
+    ./backend-project/*/tests/__init__.py: F401
```

Comment:
Jeżeli jest to konieczne (wieczorem muszę na to popatrzeć) to bm zrobił to raczej przez komentarze w pliku. Łatwiej ogarnąć, gdy rzeczy są blisko siebie.